src/components/ControlPlane/GitRepositories.tsx

```diff
@@ -152,21 +162,32 @@ export function GitRepositories() {
     }) ?? [];
 
   return (
-    <Panel
-      fixed
-      header={
-        <Toolbar>
-          <Title>{t('common.resourcesCount', { count: rows.length })}</Title>
-          <YamlViewButton variant="resource" resource={data as unknown as Resource} />
-          <ToolbarSpacer />
-        </Toolbar>
-      }
-    >
-      <>
-        <ConfiguredAnalyticstable columns={columns} isLoading={isLoading} data={rows} />
-        <ErrorDialog ref={errorDialogRef} />
-      </>
-    </Panel>
+    <>
+      <Panel
+        fixed
+        header={
+          <Toolbar>
+            <Title>{t('common.resourcesCount', { count: rows.length })}</Title>
+            <YamlViewButton variant="resource" resource={data as unknown as Resource} />
+            <ToolbarSpacer />
+            <Button icon="add" onClick={() => setIsCreateDialogOpen(true)}>
+              {t('buttons.create')}
+            </Button>
+          </Toolbar>
+        }
+      >
+        <>
+          <ConfiguredAnalyticstable columns={columns} isLoading={isLoading} data={rows} />
+          <ErrorDialog ref={errorDialogRef} />
+        </>
+      </Panel>
+
+      <CreateGitRepositoryDialog
+        isOpen={isCreateDialogOpen}
+        namespace={rows[0]?.item?.metadata?.namespace || 'default'}
```

Comment:
Interesting idea, taking the first item as a default value. I discussed with colleagues: There’s a follow-up task where we will improve this by showing a drop-down field with the available namespaces (https://github.com/openmcp-project/backlog/issues/150). In the meantime, let's keep it simple and just use the hard-coded "default" namespace as default, i.e. no need to pass anything here.


```suggestion
```